queries.sql

```diff
@@ -14,4 +14,33 @@ SELECT name FROM animals WHERE neutered IS TRUE;
 
 SELECT name FROM animals WHERE name NOT IN ('Gabumon');
 
-SELECT name FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT name FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species='unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species='pokemon' WHERE species IS NULL;
+COMMIT;
```

Comment:
- [ ] You should verify that change was done after commit as requested in the project requirements

> - Commit the transaction.
> - Verify that change was made and persists after commit.